webrtc.html

```diff
@@ -32,9 +32,19 @@
 
     <p>The following features are marked as at risk:</p>
     <ul>
-      <li>The value <a href="#dom-rtcrtcpmuxpolicy-negotiate"><code>negotiate</code></a> of RTCRtcpMuxPolicy.</li>
-      <li>The <code>encodings</code> attribute of <code><a>RTCRtpReceiveParameters</a></code>.</li>
+      <li>The <code>oauth</code> value of <code>RTCIceCredentialType</code> and the
+      <code>RTCOAuthCredential</code> dictionary.</li>
+      <li>The <code>voiceActivityDetection</code> member of the <code><a>RTCOfferAnswerOptions</a></code> dictionary.</li>
       <li>The <code>getDefaultIceServers()</code> method of <code><a>RTCPeerConnection</a></code>.</li>
+      <li>The <code>RTCPriorityType</code> enum and the <code>priority</code> member of the
+      <code>RTCRtpSendParameters</code> dictionary.</li>
```

Comment:
Should this be updated to include maxFramerate now that we merged that PR as well?